raw_data/preprocessed/africa/seychelles_english.txt

```diff
@@ -0,0 +1,91 @@
+
+(31) Any person who: [a] destroys , pulls down, removes, injures or defaces any pillar, post or stake fixed by the Authority or any notice or other matter put up, inscribed or placed by or under the order of the Authority; or [b] obstructs any person acting under the orders or directions of the Authority from exercising the powers and performing the functions under this Act; or [c] damages any works or property belonging to the Authority, or [d] fails to furnish to any officer or other employee of the Authority a report or any information required by the officer or employee for the purpose of this Act, or [e] fails to intimate the occurrence of any accident or other unforeseen act to the Authority, or [f] in giving a report or any information which the person is required to give under this Act, makes a statement which is false or misleading in any material particular, is guilty of an offence and is liable on conviction to imprisonment for 6 months and a fine of R10.000.
+
+(32) In addition to imposing a penalty for an offence arising under the provisions of this Act, the Court may order the person convicted to compensate for any loss or damage to the environment and to take such steps as may be specified in the order and within such time as may be specified, to pay damages and to prevent, control, abate or mitigate any harm to the environment caused by the commission of the offence or to prevent the continuance or recurrence of the offence.
+
+(33) [1] Where any offence under this Act has been committed by a company, every person who, at the time of offence was committed, was in charge of, and was responsible to, the company for the conduct of the business of the company, as well as the company is guilty of the offence and liable to be proceeded against and punished accordingly: Provided that nothing contained in this sub-section shall render any such person liable to any punishment provided in this Act, if the person proves that the offence was committed without the knowledge of that person or that the person exercised all due diligence to prevent the commission of such offence. [2] Where an offence under this Act has been committed with the consent or connivance of, or is attributable to any neglect on the part of any director, manager, secretary or other officer of the company, such director, manager, secretary or other officer is guilty of that offence and shall be liable to be proceeded against and punished accordingly. [3] For the purposes of this section: [a] "company" means any body corporate and includes a partnership or other association of individuals; [b] "director" in relation to a partnership, means a partner in the firm.
```

Comment:
Please replace `the time of offence was committed` with `the time the offence was committed`. Also, please replace `accordingly: Provided` with `accordingly, provided`.